src/entry.rs

```diff
@@ -130,50 +161,13 @@ impl Resolver {
         } else {
             None
         };
-
```

Comment:
We'll be able to remove the `Err` part for `fn load_entry_uncached(&self, path: &Path) -> RResult<Entry> {`